packages/compiler-core/__tests__/__snapshots__/codegen.spec.ts.snap

```diff
@@ -79,8 +79,9 @@ return function render() {
     const _component_Foo = _resolveComponent(\\"Foo\\")
     const _component_bar_baz = _resolveComponent(\\"bar-baz\\")
     const _component_barbaz = _resolveComponent(\\"barbaz\\")
-const _directive_my_dir = _resolveDirective(\\"my_dir\\")
```

Comment:
@yyx990803 this pr, fix the indentation, though it add some empty line with leading spaces on other snapshots.